tests/product/test_topology.py

```diff
@@ -58,26 +58,6 @@ def test_topology_show(self):
         expected = normal_topology
         self.assertEqual(expected, actual)
 
-    @docker_only
```

Comment:
topology show doesn't connect to any remote hosts, so this is a weird thing to test.